Tools/lib/python/eupath/RnaSeqEupathExporter.py

```diff
@@ -5,46 +5,90 @@
 import os
 import re
 
-class RnaSeqExport(EupathExporter.Export):
-
+class RnaSeqExporter(EupathExporter.Exporter):
+
+    """
+INPUT
+    tuple format is: [filepath, samplename, refgenome_key, suffix]
+
+    suffixes are:  bw or txt (for fpkm files)
+    
+    type specific args:  
+      strand param: 'stranded' or 'unstranded'
+      for each sample, a list of tuple pairs, one txt tuple and one bw tuple: 
+        - hopefully all tuples agree on ref genome
+        - hopefully matching bw and txt files
+        - if stranded, then sample has two pairs of tuples, one per strand
+          - txt:
+            - first tuple file name and sample name contain 'forward' in the name (sense)
+            - second tuple file name and sample name contain 'reverse' in the name (antisense)
+          - bw:
+            - first tuple is strand one
+            - second tuple is strand two
+
+OUTPUT
+  files given cannonical names:
+     unstranded: sample.suffix (with sample name cleaned of icky characters)
+     stranded:  
+       -SAMPLE_NAME.forward.bw
+       -SAMPLE_NAME.reverse.bw
+       -SAMPLE_NAME.one.txt
+       -SAMPLE_NAME.two.txt
```

Comment:
are the `.txt` and `.bw` files in this example reversed, or do they get reversed by the process?